packages/base/src/Bootstrap.ts

```diff
@@ -24,7 +24,7 @@ export function isAPlugin(packageName?: string) {
     }
 
     const packageJson = require(packageName + "/package.json") as Object;
-    if (!packageJson.hasOwnProperty("corejam")) return false;
+    if (!Object.prototype.hasOwnProperty.call(packageJson, "corejam")) return false;
```

Comment:
Which lint rule? That is ugly as hell. Could you use "key in obj"?

Reply:
https://eslint.org/docs/rules/no-prototype-builtins

Reply:
and yea agreed that it looks ugly as hell. Im happy to just keep the rule off